articles/roadmap-preview-october-2025.md

```diff
@@ -0,0 +1,33 @@
+# Roadmap preview, October 2025
+
+<div purpose="embedded-content">
+   <iframe src="TODO" allowfullscreen></iframe>
+</div>
+
+The Fleet roadmap is set for fall 2025. Watch the video above for a walkthrough, or continue reading for the highlights.
+
+In the next 3 months, Fleet will ship...
+
+- 👔 Customize first-time Linux and Windows setup
+- 📄 Deploy certificates for Wi-Fi and VPN on company-owned Linux & Windows computers and personal mobile devices (iOS/iPadOS, Android)
+- 🔧 Self-service scripts on Linux, macOS, and Windows
+- 🛡️ Integration with Okta for conditional access on macOS
```

Comment:
@rachaelshaw is Okta conditional access just macOS in Q4?

Reply:
macOS only for now, TBD if Linux/Windows are Q4 2025 or Q1 2026.